resteasy-upgrade-guide/reference/en/en-US/modules/Miscellaneous.xml

```diff
@@ -121,4 +121,49 @@ ResteasyProviderFactory.getInstance().registerProvider(POJOConverterProvider.cla
    </para>
    
    </section>
+
+   <section>
+      <title>Matching Requests to Resource Methods</title>
+
+      <para>
+         Improvements and corrections have been made in Resteasy 3 to the implementation of
+         the matching rules as defined in the JAX-RS 2.0 specification.  In particular
+         a change for handling an ambiguous uri on a sub-resource method and
+         sub-resource locator was made.  In Resteasy 2 it was possible for a
+         sub-resource locator to execute successfully even when there was also a sub-resource
+         with the same uri.  This was incorrect behavior according to the specification.
+         In Resteasy 3 when there is an ambiguous uri for a sub-resource and a
+         sub-resource locator, calling the sub-resource will be successful, but calling
+         the sub-resource locator will result in an HTTP, 405 Method Not Allowed error.

```

Comment:
@rsearls , I'm uncomfortable with the phrase "calling the sub-resource locator". How does one call a sub-resource locator? You have to send a URI that matches the sub-resource locator but which does not match a sub-resource method (which would "mask" the sub-resource locator). So, when a sub-resource method and sub-resource locator have the same URI, it isn't possible to call the sub-resource locator. So, I can't make sense of "calling the sub-resource locator will result in an HTTP, 405 Method Not Allowed error". I would say that it is the calling of the sub-resource method with the wrong verb is what leads to the 405 status. WDYT?